src/ensembl/src/shared/accordion/components/Accordion.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import DisplayName from '../helpers/DisplayName';
+import { DivAttributes } from '../helpers/types';
+import { Consumer, Provider } from './AccordionContext';
+import { UUID } from './ItemContext';
+import defaultStyles from '../css/Accordion.scss';
+import classNames from 'classnames';
+
+type AccordionProps = Pick<
+  DivAttributes,
+  Exclude<keyof DivAttributes, 'onChange'>
+> & {
+  preExpanded?: UUID[];
+  allowMultipleExpanded: boolean;
+  allowZeroExpanded: boolean;
+  extendDefaultStyles: boolean;
+  onChange?(args: UUID[]): void;
+};
+
+const Accordion = (props: AccordionProps) => {
+  const {
+    preExpanded,
+    allowMultipleExpanded,
+    allowZeroExpanded,
+    className,
+    extendDefaultStyles,
+    onChange,
+    ...rest
+  } = props;
+
+  const renderAccordion = (): JSX.Element => {
+    let styles = className;
+
+    if (extendDefaultStyles) {
+      styles = classNames(defaultStyles.accordionDefault, className);
+    }
+
+    return (
+      <div data-accordion-component="Accordion" className={styles} {...rest} />
+    );
+  };
+
+  return (
+    <Provider
+      preExpanded={preExpanded}
+      allowMultipleExpanded={allowMultipleExpanded}
+      allowZeroExpanded={allowZeroExpanded}
+      onChange={onChange}
+    >
+      <Consumer>{renderAccordion}</Consumer>
+    </Provider>
+  );
+};
+
+Accordion.diplayName = DisplayName.Accordion;
```

Comment:
Given that display name of the component is inferred automatically from the name of the variable:

![image](https://user-images.githubusercontent.com/6834224/55414681-7ab02c00-5563-11e9-8204-2d135a94604c.png)

(https://reactjs.org/docs/react-component.html#displayname)

— why are we setting displayName explicitly? It does not differ from the name of components, as far as I can tell.

Reply:
Removed the setting of `displayName` from all the components.